templates/terraform/command-center-project/variables.tf

```diff
@@ -32,3 +32,8 @@ variable vault_prefix {
   type = string
   description = "Path prefix for secrets written to Vault."
 }
+
+variable service_account_id {
+  type = string
+  description = "Name for the google service account"
```

Comment:
I think I wrote the ticket saying to make this a variable, but the more I've thought about it the more I've warmed up to the idea of just hard-coding a consistent name within the `command-center-project` module. What do others think?

Reply:
Seeing as its a variable for the command center, hard-coding seems fine to me.

Reply:
I agree that it doesn't need its own variable since we aren't using it anywhere else at the moment. Do you know if we will need to use it elsewhere in the future?

Reply:
I think if it's only being used in that module and only referenced that one time, I think it's fine to hard code it. We could always make it a variable down the line if we need to reference it elsewhere